tools/sggolangcilint/tools.go

```diff
@@ -28,15 +28,29 @@ func Command(ctx context.Context, args ...string) *exec.Cmd {
 	return sg.Command(ctx, sg.FromBinDir(name), args...)
 }
 
-// Run GolangCI-Lint in every Go module from the root of the current git repo.
-func Run(ctx context.Context, args ...string) error {
-	defaultConfigPath := sg.FromToolsDir("golangci-lint", ".golangci.yml")
-	if err := os.MkdirAll(filepath.Dir(defaultConfigPath), 0o755); err != nil {
-		return err
+func defaultConfigPath() string {
+	return sg.FromToolsDir(name, ".golangci.yml")
+}
+
+func CommandInModule(ctx context.Context, directory string, args ...string) *exec.Cmd {
```

Comment:
Maybe `RunInDirectory`? What do you think?

Reply:
Or maybe `CommandInDirectory` (I've realized that this does not return an error :sweat_smile:

Reply:
Yeah, `CommandInDirectory` sounds better!